### DISCUSSION TITLE
When I try to update my 14 mb qsvlite with the `--update` option, it downloads a whopping 190 mb file. How come?

### DISCUSSION THREAD

**Discussion by jqnatividad** [dathere/qsv#1714] [FAQ]
That's because for each platform, **_ALL the binary variants of qsv_** are in one zip archive (see #1713).

So even if you're just updating qsvlite (14 mb in qsv 0.124.1), it needs to download the whole archive of the latest version (in this case, 190 mb for [qsv-0.125.0-x86_64-unknown-linux-gnu.zip](https://github.com/jqnatividad/qsv/releases/download/0.125.0/qsv-0.125.0-x86_64-unknown-linux-gnu.zip)) to extract the qsvlite variant.

And no need to worry about the big archive hanging around, `qsvlite --update` automatically cleans after itself and deletes the 190 mb file after an update.

**Comment by ondohotola** [comment]
Would be nice, actually, to have the ZIP hang around, so one doesn’t have to downloads multiple times for every local machine. Could that be added by way of a flag, perhaps?
